LTC - 37. Sudoku Solver

```diff
@@ -0,0 +1,45 @@
+/*Recursive approch with trial and error method
+ *Runtime: 17 ms, faster than 48.86% of Java online submissions for Sudoku Solver.
+ *Memory Usage: 41.9 MB, less than 33.21% of Java online submissions for Sudoku Solver.
+ *
+ *Write a program to solve a Sudoku puzzle by filling the empty cells
+ *
+ */
+class Solution {
+    public void solveSudoku(char[][] board) {
+        solve(board);
+    }
+    public boolean solve(char[][] board){
+        for(int i=0; i<9; i++){
```

Comment:
Very good explanatory comments. @manjunathdyavanna  
Try to add time and space complexity in your solutions